tool/jt.rb

```diff
@@ -1775,13 +1789,19 @@ def build_native_image(*options)
       File.write('mx.substratevm/env', "JAVA_HOME=#{java_home}\n") if jvmci
 
       mx 'build'
-      mx 'fetch-languages', '--Language:llvm', '--Language:ruby'
+      mx 'fetch-languages',
+         '--Language:llvm', '--Language:ruby',
+         '--Tool:chromeinspector', '--Tool:profiler'
```

Comment:
I think this might break the native distribution when we no longer depend on tools (#1258), since the CI uses `mx build`.
To ensure it works, please rebase this PR or #1258 after merging one of them.

Reply:
sure, I thought about that too.

Reply:
I think this works because SVM depends on tools currently. Probably good enough for now.

Reply:
It calls `jt build` which adds the dynamic import of tools so they should be always available.

Reply:
Oh right, of course.